app/policies/division_policy.rb

```diff
@@ -1,22 +1,34 @@
 class DivisionPolicy < ApplicationPolicy
   def index?
-    division_member_or_admin(division: @record)
+    # Require admin role on at least one division to allow access to index view
+    any_division_admin?
   end
 
   def show?
     division_member_or_admin(division: @record)
   end
 
   def create?
-    division_admin(division: @record)
+    # Need to let record with missing parent pass through the policy check so a validation
+    # message can be presented to the user.
+    # Todo: Confirm if there a better approach here?
+    return true unless @record.parent
+    division_admin(division: @record.parent)
```

Comment:
I would put this logic in the controller using skip_authorization or so. Putting it here risks side effects if this policy is used somewhere else.